RELEASE-LEADS.md

```diff
@@ -48,13 +48,15 @@ This roster is seeded with all approvers from Eventing workgroups.
 | v0.17   | 2020-08-18   | yanweiguo      | Harwayne        | -           | 2020-08-11 |
 | v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott        | 2020-08-19  | 2020-09-22 |
 | v0.19   | 2020-11-10   | julz           | n3wscott        | 2020-09-30  | 2020-11-03 |
-| v0.20   | 2020-12-22   | nak3           | slinkydeveloper | 2020-11-11  | 2020-12-15 |
-| v0.21   | 2021-02-02   | mattmoor       | lionelvillard   | 2020-12-23  | 2021-01-26 |
-| v0.22   | 2021-03-16   | markusthoemmes | evankanderson   | 2020-02-03  | 2021-03-09 |
-| v0.23   | 2021-04-27   | tcnghia        | vaikas          | 2020-03-17  | 2021-04-20 |
-| v0.24   | 2021-06-08   | dprotaso       | matzew          | 2020-04-28  | 2021-06-01 |
-| v0.25   | 2021-07-20   | vagababov      | grantr          | 2020-06-09  | 2021-07-13 |
-| v0.26   | 2021-08-31   | JRBANCEL       | ...             | 2020-07-21  | 2021-08-24 |
+| v0.20   | 2020-01-05   | nak3           | slinkydeveloper | 2020-11-11  | 2020-12-15 |
```

Comment:
2020-01-05?

Reply:
2020-01-06 it's a religious holiday in some eu places, can we move it to 2020-01-07?

Reply:
@slinkydeveloper As per [the instruction](https://github.com/knative/pkg/blob/master/RELEASE-LEADS.md#1-day-prior-to-the-release), `1 day prior to the release` and `Day of the release` have some tasks, so 2021-01-05 is better than 2021-01-07?
We may have extra task if there are any troubles on `2021-01-06` :smile: but the US folks are supposed to work on that day.

Reply:
> We may have extra task if there are any troubles on 2021-01-06 smile but the US folks are supposed to work on that day.

That's my worry, i won't be online for sure on 01-06 because it's holiday in italy and i'm worried that eu folks might be offline the 2 days before too (4 and 5)